apps/fipsinstall.c

```diff
@@ -81,6 +82,41 @@ const OPTIONS fipsinstall_options[] = {
     {NULL}
 };
 
+typedef struct {
+    unsigned int self_test_onload : 1;
+    unsigned int conditional_errors : 1;
+    unsigned int security_checks : 1;
+    unsigned int tls_prf_ems_check : 1;
+    unsigned int drgb_no_trunc_dgst : 1;
+} FIPS_OPTS;
+
+/* Pedantic FIPS compliance */
+static const FIPS_OPTS pedantic_opts = {
+    1,      /* self_test_onload */
+    1,      /* conditional_errors */
+    1,      /* security_checks */
+    1,      /* tls_prf_ems_check */
+    1,      /* drgb_no_trunc_dgst */
```

Comment:
Maybe leave the last ',' out on these tables to avoid accidents when adding additional options..

Reply:
I left it in to reduce clutter in the diffs when adding additional options…

Reply:
Given that fields can be added anywhere, I'm not sure how leaving the comma off helps.

Reply:
Doesnt help with the order, only the size if you forget to add one?

Reply:
It won't help there.  Extra fields will be zero filled regardless of the comma or not.
C doesn't have a way to mandate initialisation of all fields.